src/firefly/java/edu/caltech/ipac/firefly/server/util/QueryUtil.java

```diff
@@ -95,6 +96,38 @@ public static TableServerRequest convertToServerRequest(HttpServletRequest req)
         return retval;
     }
 
+    public static TableServerRequest convertToServerRequest(String str) {
```

Comment:
What is the difference between QueryUtil.convertToServerRequest and TableServerRequest.parse? When should we use each?